### PR TITLE
First Capstone Project - Conference Page

### PR DESCRIPTION
**Pull request by mwismann**
### First Capstone Project - Conference Page: FreeForAil 😎 
**In this branch I worked on the following features of the project:**

- Selected a Topic for my project ✅ 
- Added HTML files for the home and about pages ✅
- Worked with Tailwind Framework ✅ 
- Stuck to the original behance design as much as possible ✅ 
- Made some minor necessary adjustments (e.g.: color contrast) to improve the accessibility of the website ✅ 
- Made the site responsive for all screen sizes ✅ 

**> Added the following link interactions 🤠 :** 

1. The home page has a link in the menu to the about page ✅ 
2. The logo in the header links to the home page ✅ 

**> Added the following mobile menu interactions 🤠 :**

1. When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page ✅
2. When the user interface with the mobile menu links, the mobile menu closes ✅  
3. The mobile menu has a close (X) button that closes the menu ✅ 

**> Made the page dynamic 🤠 :**

1. The section "Featured speakers" is created dynamically in JavaScript.
2. Declared a JavaScript variable to store the speakers' data and used it inside of the function I called to populate the Featured Speakers Section✅ 

- Created a professional README for the repo ✅ 
- Recorded a loom walkthrough with the link inside the README ✅ 
- Gave appropriate credit to the author, Cindy Shin, for the original design inside the README✅

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+                    <li><a href="#" class="text-xs xl:text-base">News</a></li>
+                </ul>
+            </nav>
+        </section>
+        <div class="py-2 px-4 text-3xl bg-[#dfdfdf] md:hidden">
+            <button class="open-mobile"><i class="las la-bars"></i></button>
+            <div class="mobile-menu hidden relative">
+                <ul class="fixed inset-0 pt-6 px-4 flex flex-col justify-center items-center bg-[#dfdfdf]">
+                    <button class="close-mobile absolute left-4 top-2"><i class="las la-times"></i></button>
+                    <li class="close-mobile w-full py-10 border-y-2 text-center"><a href="about.html" class="highlight text-sm">About</a></li>
```
Comment (huseyinbiyik) [inline about.html]: Good job with the mobile menu 👍 Let's add a link to the Home page so when we on mobile view and we are on the About, we can go to Homepage with that link.

Screenshot 2023-02-08 at 10 55 34 PM

File: index.html [hunk 2]
```diff
@@ -7,33 +7,145 @@
...
+                    <li><a href="#partners" class="text-xs xl:text-base">Sponsor</a></li>
+                    <li><a href="#" class="text-xs xl:text-base">News</a></li>
                 </ul>
-                <button><i class="las la-bars"></i></button>
-                <button><i class="las la-times"></i></button>
             </nav>
         </section>
+        <div class="py-2 px-4 text-3xl bg-[#dfdfdf] md:hidden">
+            <button class="open-mobile"><i class="las la-bars"></i></button>
+            <div class="mobile-menu hidden relative">
```
Comment (huseyinbiyik) [inline index.html:44]: Mobile button functionality works great 👍  But let's follow the provided design. Here the menu bar needs to be transparent as seen below. Kindly fix that.
Screenshot 2023-02-08 at 10 55 15 PM
Screenshot 2023-02-08 at 11 31 20 PM

File: README.md [hunk 3]
```diff
@@ -1 +1,160 @@
...
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- https://mwismann.github.io/first-capstone/
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Loom Walkthrough <a name="loom-walk"></a>
+
+- https://www.loom.com/share/ef6b4080224a4db99c282927cf754e31
```
Comment (huseyinbiyik) [inline README.md]: Great video 👍 , it matches all the requirements except one. Kindly record a video that has a length of a maximum of 5 minutes. 
Screenshot 2023-02-08 at 11 17 10 PM

File: index.html [hunk 4]
```diff
@@ -7,33 +7,145 @@
...
+        <ul class="mt-4 px-8 flex flex-wrap justify-center items-center gap-5 lg:mt-3 lg:gap-9 xl:mt-10 xl:gap-20">
+            <li><a href="" class="text-xl text-[#dfdfdf] font-bold md:text-2xl xl:text-[2.5rem]">OpenAI</a></li>
+            <li><a href="" class="text-xl text-[#dfdfdf] font-bold md:text-2xl xl:text-[2.5rem]">Google</a></li>
+            <li><a href="" class="text-xl text-[#dfdfdf] font-bold md:text-2xl xl:text-[2.5rem]">Meta Platforms</a></li>
+            <li><a href="" class="text-xl text-[#dfdfdf] font-bold md:text-2xl xl:text-[2.5rem]">NVIDIA</a></li>
+            <li><a href="" class="text-xl text-[#dfdfdf] font-bold md:text-2xl xl:text-[2.5rem]">Apple</a></li>
+        </ul>
+    </section>
 
+    <footer class="py-12 bg-[#dfdfdf] flex justify-evenly items-center gap-4 lg:px-40 lg:justify-start lg:gap-16 xl:py-24 xl:px-60">
```
Comment (huseyinbiyik) [inline index.html]: Great job using semantic footer element. 👍  Let's align the element inside the footer into the center as seen below.
Screenshot 2023-02-08 at 11 37 54 PM
Screenshot 2023-02-08 at 11 39 00 PM

Comment (huseyinbiyik) [inline index.html]: Please fix the color of the footer element and make it white on the mobile view like below.

Screenshot 2023-02-08 at 11 44 33 PM
Screenshot 2023-02-08 at 11 45 05 PM

File: index.html [hunk 5]
```diff
@@ -7,33 +7,145 @@
...
+        <div class="logo-container">
+            <a href="index.html" class="text-lg font-bold lg:text-3xl xl:text-4xl"><i class="las la-brain highlight"></i> FreeFor<span class="highlight">Ail</span></a>
+            <span class="sr-only">FreeForAil Logo</span>
+        </div>
+        <div id="copyright">
+            <span class="block text-xs md:inline xl:text-base">2023 FreeForAil Corp.</span>
+            <span class="block text-xs md:inline xl:text-base">All Rights Reserved.</span>
+        </div>
+    </footer>
```
Comment (huseyinbiyik) [inline index.html:149]: Kindly make sure the background color for the desktop view is white for the your homepage like on the design.
Screenshot 2023-02-08 at 11 49 08 PM
Screenshot 2023-02-08 at 11 49 27 PM

File: styles/styles.css [hunk 6]
```diff
@@ -0,0 +1,26 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
+  color: var(--main-dark);
+
+  --main-dark: #191d25;
+  --main-orange: #e4250f;
+  --main-gray: #dfdfdf;
```
Comment (huseyinbiyik) [inline styles/styles.css]: Kindly fix your color values as seen in the design guidelines.
Screenshot 2023-02-08 at 11 57 37 PM

File: index.html [hunk 7]
```diff
@@ -7,33 +7,145 @@
...
+    </section>
     
+    <section id="main-program" class="pt-8 bg-[#191d25] text-center md:pb-12 lg:pb-16 xl:min-h-[60vh]">
+        <h2 class="text-xl text-white font-semibold xl:text-3xl">Main Program</h2>
+        <div class="flex justify-center">
+            <i class="las la-minus text-2xl text-[#ec5242]"></i>
+            <i class="las la-minus text-2xl text-[#ec5242] ml-[-0.5rem]"></i>
+        </div>
+    
+        <div class="program-grid mt-3 px-3 flex flex-wrap gap-2 md:px-12 md:justify-center md:gap-1 md:flex-nowrap lg:px-36 xl:mt-6 xl:px-60">
```
Comment (huseyinbiyik) [inline index.html]: Good job with wrapping semantic section element. Let's wrap our card with semantic article tag to make it more semantic and SEO friendly.

File: scripts/config.js [hunk 8]
```diff
@@ -1,2 +1,94 @@
...
+
+    <div class="speakers-grid pl-2 pr-3 grid grid-cols-1 grid-rows-6 gap-8 md:grid-cols-2 md:grid-rows-3 md:gap-x-2 md:px-12 lg:px-36 xl:mt-8 xl:px-60 xl:gap-y-24">
+    </div>`;
+
+  speakersSection.insertAdjacentHTML('beforeend', speakersContent);
+
+  const speakersGrid = document.querySelector('.speakers-grid');
+
+  speakersData.forEach((speaker) => speakersGrid.insertAdjacentHTML('beforeend',
+    `<div class="speaker-card relative flex gap-16 md:gap-20 lg:gap-24 xl:gap-20">
```
Comment (huseyinbiyik) [inline scripts/config.js]: Kindly use semantic article tag instead of div for each speaker card to make your web page more semantic and SEO friendly.

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @mwismann,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Speakers section created dynamically 👍 
- Responsive design 👍 
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- Commit messages are meaningful and written in imperative tense 📝
- PR description is descriptive 📝

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
**Hi @mwismann 

You did great. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights

✔️ Descriptive PR
✔️ Nice UI
✔️ Well done implementing the requested changes 

## Optional suggestions



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Pull request merged**